certified-connectors/Talkdesk/apiDefinition.swagger.json

```diff
@@ -2920,6 +2921,135 @@
         ]
       }
     },
+    "/ccaas/users/status": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {}
```

Comment:
@StefanCRT22 empty schema is not allowed. Please define a proper schema.

Reply:
Hi @Amjed-Ayoub 
The reason that I didn't added the schema is because I've tested the connector using this schema:

```
"200": {
		"description": "Data response for this action.",
		"schema": {
		"type": "array",
                "items": {
                    "type": "object",
                    "properties": {
                      "id": {
                        "type": "string",
                        "description": "User id.",
                        "title": "Id"
                      },
                      "status": {
                        "type": "string",
                        "description": "User's current status.",
                        "title": "Status"
                      }
                    }
                  }
		}
          }
```
But it seems that when I run the Power Automate it gives me this error:
![image](https://user-images.githubusercontent.com/89248341/202390324-ec8b70b0-248d-45bb-8b10-839a32477181.png)

And when I test the action directly from the custom connector it's working fine:
![image](https://user-images.githubusercontent.com/89248341/202390761-6bccc634-77d3-4c2b-967f-ca39ac9b447d.png)

I don't understand why it's happening this.
Please let me know if you want me to add the schema to the swagger so you can test it as well.

Reply:
@StefanCRT22 this action is "GetUserStatus", and you are showing the schema for "GetUserPresence" which you are deprecating in this update, and the schema type is object.
 "200": {
            "description": "User Presence.",
            "schema": {
              "type": "object",
              "properties": {
                "id": {
                  "type": "string",
                  "description": "A unique id for the event.",
                  "title": "Id"
                },........

May I ask you please to elaborate more? is it possible to define a proper schema for "GetUserStatus" and test it please?

Please let me know if you have any questions.

Thank you so much.

Reply:
Sorry about that, I was testing the schema without changing the name of the action.
I just updated the schema for GetUserStatus with this one
```
"200": {
		"description": "Data response for this action.",
		"schema": {
		"type": "array",
                "items": {
                    "type": "object",
                    "properties": {
                      "id": {
                        "type": "string",
                        "description": "User id.",
                        "title": "Id"
                      },
                      "status": {
                        "type": "string",
                        "description": "User's current status.",
                        "title": "Status"
                      }
                    }
                  }
		}
          }
```

 and when I test it from flow I get this error
![image](https://user-images.githubusercontent.com/89248341/202668587-f7a1a174-9440-40cd-9884-83e5fa029e85.png)

and when I test the same action from the custom connector editor everything works fine
![image](https://user-images.githubusercontent.com/89248341/202669144-e7f4bdea-b273-4bc5-b2cc-0201ccd6ca54.png)

Reply:
Hi @Amjed-Ayoub 
That API belongs to another BU and we don't have access to change it.
I've made more tests, including installing the connector in a different env and also creating another action in same connector. But the results are the same...same error.
What I've noticed is that if I change the type in the swagger, from array to object, in the Test tab of the Connector it will identify that the expected response is an array,
![Capture4](https://user-images.githubusercontent.com/89248341/203787976-b4d4d594-c136-44c0-8539-1ace442fb440.PNG)

 but in the Power Automate it will throw again an error saying that the response it's a string
![image](https://user-images.githubusercontent.com/89248341/203787890-85b20820-d7e5-4d35-9669-7b53df9093fd.png)

Maybe in Power Automate the response is converted to string? Why this difference between Test and Power Automate?

Reply:
Sorry about all this misunderstanding. I just talked with o guy from the other NU wich it's supposed to own the API and it seems that the response is a String. So now I've updated the swagger.